uuid_test.go

```diff
@@ -251,6 +251,20 @@ func TestFromBytesOrNil(t *testing.T) {
 	}
 }
 
+func TestMarshalJSON(t *testing.T) {
```

Comment:
In marshalling an error can't happen.

In unmarshalling there is already that case:
b2 := []byte("")
	u2 := UUID{}

	err = u2.UnmarshalJSON(b2)
	if err == nil {
		t.Errorf("Should return error trying to unmarshal from empty string")
	}